incubator/nodejs/image/Dockerfile-stack

```diff
@@ -35,7 +35,11 @@ COPY ./LICENSE /licenses/
 COPY ./project /project
 COPY ./config /config
 
+RUN npm install npm@latest
```

Comment:
The build image is damaged, and this is just restoring it to its upstream state. That seems unwise to me.  How to prove to yourself that the container when run by the appsody build env is _not_ the same as the upstream image distributed by redhat: https://github.com/kabanero-io/collections/issues/274#issuecomment-591589605

registry.access.redhat.com/ubi8/nodejs-12:1-27 only has one node_modules/tar, except in the appsody build image... where it has two (one of which is broken). I think that's concerning.

If its not worth investigating, consider that this will be force-update npm past what is distributed in the image, you might consider doing `npm install -g npm@6.12.1` (https://github.com/kabanero-io/collections/issues/274#issuecomment-591601012)